lib/fog/ovirt/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fog
   module Ovirt
-    VERSION = '0.1.2'
+    VERSION = '0.1.2'.freeze
```

Comment:
Please move this to a separate pr, there are now merge conflicts because the version was bumped before this pr was merged.

Reply:
sorry on that :'( removed the freeze

Reply:
The freeze was a good change, just didn't want it to affect the rest of the pr.
It's already suggested in: https://github.com/fog/fog-ovirt/pull/5/files#diff-73b69b006fbdd314071398980a75b3eaR3